result/lldpneighbors.yaml

```diff
@@ -0,0 +1,155 @@
+components:
+  schemas:
+    LldpNeighbors.State.Request:
+      description: >-
+        The request to retrieve LLDP per interface neighbor information.
+      type: object
+      properties:
+        lldp_names:
```

Comment:
"lldp_names" or "lldp_interfce_names" or "interface_names" ?

Reply:
changed to lldp_interface_names